### PR TITLE
Adds stress balls to loadout/psych office

### PR DESCRIPTION
**Pull request by Llywelwyn**
- rscadd: "Added colourable stress balls to the loadout."
  - rscadd: "Added a box of stress balls to the psychiatrist's locker."

image
image

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!review

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1475,3 +1476,39 @@
...
+		if(I_HELP)
+			user.visible_message(SPAN_NOTICE("[SPAN_BOLD("\The [user]")] plays with \the [src]!"), SPAN_NOTICE("You play with \the [src]!"))
+		if(I_DISARM)
+			user.visible_message(SPAN_NOTICE("[SPAN_BOLD("\The [user]")] squeezes \the [src]!"), SPAN_NOTICE("You squeeze \the [src]!"))
+			volume = 30
+		if(I_GRAB)
+			user.visible_message(SPAN_NOTICE(SPAN_BOLD("\The [user] pinches \the [src]!")), SPAN_NOTICE(SPAN_BOLD("You pinch \the [src]!")))
+			volume = 40
+		if(I_HURT)
+			user.visible_message(SPAN_WARNING(SPAN_BOLD("\The [user] crushes \the [src]!")), SPAN_WARNING(SPAN_BOLD("You crush the [src]!")))
```
Comment (FluffyGhoster) [inline code/game/objects/items/toys.dm:1511]:
```suggestion
			user.visible_message(SPAN_WARNING(SPAN_BOLD("\The [user] crushes \the [src]!")), SPAN_WARNING(SPAN_BOLD("You crush \the [src]!")))
```

**Review by FluffyGhoster — COMMENTED** [review]
LGTM

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Llywelwyn) [reply code/game/objects/items/toys.dm:1511]: woe, there's always one

**Review by FluffyGhoster — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]
soul

**Pull request merged**